api/src/main/java/org/apache/cloudstack/api/command/admin/account/DeleteAccountCmd.java

```diff
@@ -92,7 +93,8 @@ public String getEventType() {
 
     @Override
     public String getEventDescription() {
-        User user = _responseGenerator.findUserById(getId());
+        List<User> users = _accountService.getAccountUsers(_accountService.getAccount(getId()));
+        User user = users.isEmpty() ? null : users.get(0);
```

Comment:
The event description is wrong, the account is being deleted it may not even mention the users and simply say removing all users under the account.